packages/page-objects/src/index.ts

```diff
@@ -55,6 +55,8 @@ export * from './components/sidebar/extensions/ExtensionsViewItem';
 export { ScmView, ScmProvider, ScmChange } from './components/sidebar/scm/ScmView';
 export * from './components/sidebar/scm/NewScmView';
 export * from './components/sidebar/debug/DebugView';
+export * from './components/sidebar/tree/debug/CallStackItem';
```

Comment:
it would look nicer to have it grouped under same path `./components/sidebar/tree/debug` which ends at line 52